src/views/ResourcePage.jsx

```diff
@@ -1,13 +1,22 @@
 import React, { Fragment } from 'react';
-import Preview from '../components/Preview';
+import { Helmet } from 'react-helmet';
 
+import Header from '../components/Header';
+import ResourceContent from './ResourceContent';
+import Footer from '../components/Footer';
 
 const Markup = (props) => (
   <Fragment>
-    <h1>Resource Page</h1>
-    <Preview>
-      {JSON.stringify(props, 0, 2)}
-    </Preview>
+
+    <Helmet>
```

Comment:
Also can you look at removing these and rather using SVG icons and using https://www.npmjs.com/package/gatsby-plugin-web-font-loader?

Reply:
Sure, will do.